rllib/agents/sac/apex.py

```diff
@@ -41,6 +69,243 @@
 # __sphinx_doc_end__
 # yapf: enable
 
+
+class LocalVanillaReplayBuffer(LocalReplayBuffer):
```

Comment:
What is the difference here with the LocalReplayBuffer beside not including the weights and the indices in the Batch?

Reply:
The three changes are
1. PrioritizedReplayBuffer vs VanillaReplayBuffer
2. SampleBatch containing different elements
3. Stats not containing update_priorities time.